src/components/ui/Card.js

```diff
@@ -0,0 +1,9 @@
+import classes from './Card.module.css';
+
+function Card(props) {
```

Comment:
I'm not sure if this needs to be its own element, however I like the use of `children` and you don't need to change this (just something to bear in mind next time).